shared/src/commonMain/kotlin/org/hyperskill/app/notification_onboarding/presentation/NotificationsOnboardingFeature.kt

```diff
@@ -0,0 +1,24 @@
+package org.hyperskill.app.notification_onboarding.presentation
+
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticEvent
+
+object NotificationsOnboardingFeature {
+    object State
+
+    sealed interface Message {
+        object AllowNotificationClicked : Message
+        object RemindMeLaterClicked : Message
+        data class NotificationPermissionRequestResult(val isGranted: Boolean) : Message
+        object ViewedEventMessage : Message
+    }
+
+    sealed interface Action {
+
+        data class LogAnalyticsEvent(val event: HyperskillAnalyticEvent) : Action
```

Comment:
It's OK for such a small feature, but in the future it's better to separate Actions/Messages into public and internal like we do with previous features